docker_daemon/check.py

```diff
@@ -596,10 +596,12 @@ def _report_volume_count(self):
 
         attached_volumes = self.docker_client.volumes(filters={'dangling': False})
         dangling_volumes = self.docker_client.volumes(filters={'dangling': True})
-        attached_count = len(attached_volumes['Volumes'])
```

Comment:
I'd go with `len(attached_volumes.get('Volumes', [])` and keep the rest of the code as is. That way we will report `docker.volume.count` = 0 if no volume is attached (else, we might get values stuck to one after the last one is unattached).
Same for dangling_count.

In general, we need to hunt all our unsafe lookups and use the `get` method instead